docs/tutorials/java.md

```diff
@@ -115,7 +115,7 @@ Hello world.
 
```

Comment:
Can we remove this file. We had previously decided to limit our docs just to the go out of process and the go built in